Sage/carbon
# fix(menu-item): prevent setting maxWidth when items are rendered in MenuFullscreen FE-5032

fix #4961

### Proposed behaviour


Sets `maxWidth` values to `undefined` when `MenuItem`s are rendered as children of `MenuFullscreen`

![image](https://user-images.githubusercontent.com/44157880/159041416-eb5ddb1c-6b6e-46cd-b6cd-40453ed70f60.png)

### Current behaviour


`MenuItem`s apply value passed to via `maxWidth` which breaks UI layout of `MenuFulscreen`

![image](https://user-images.githubusercontent.com/44157880/159041372-91c40993-8261-4a66-a795-8979059788ef.png)

### Checklist



- [z] Storybook added or updated if required

#### QA


### Additional context



### Testing instructions

Update can be tested in linked issue sandbox

The following CodeSandbox is an example of the broken behaviour.
You can see the new behaviour by looking at the version in the comment by `codesandbox[bot]`.
